lib/fetchGlobalID.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Treasure Global ID
+ */
```

Comment:
That's a fantastic idea, also indirectly fixes https://github.com/treasure-data/td-js-sdk/pull/31#discussion_r82303894, on it.